lib/challenge.js

```diff
@@ -19,7 +19,16 @@
   // - park
 
 // Answer Starts Here
+var computer {
```

Comment:
You're missing an operator here (between `computer` and `{`) for the assignment. Without an assignment operator, this statement is not well-formed.